launcher/ui/dialogs/ResourceUpdateDialog.cpp

```diff
@@ -264,6 +264,7 @@ void ResourceUpdateDialog::checkCandidates()
 auto ResourceUpdateDialog::ensureMetadata() -> bool
 {
     auto index_dir = indexDir();
+    auto index_dir_resolver = [this](Resource* resource) { return m_resourceModel->indexDirForResource(*resource); };
```

Comment:
Not necesary a required change but I would use std::bind here to not write this lambda or give the address directly